src/app/utils/url-generator.js

```diff
@@ -9,19 +9,35 @@ export const host = () => {
 
 export const protocolAndHost = () => `${location.protocol}//${host()}`;
 
+export const newSandboxUrl = () => `/s/new`;
+
+const sandboxGitUrl = (git: {
+  repo: string,
+  branch: string,
+  username: string,
+  path: string,
+}) =>
+  'github/' + // eslint-disable-line prefer-template
+  encodeURIComponent(git.username) +
+  '/' +
+  encodeURIComponent(git.repo) +
+  '/tree/' +
+  encodeURIComponent(git.branch) +
+  '/' +
+  encodeURIComponent(git.path);
```

Comment:
does this work? I'm not sure that `path` should be encoded. The others still need to be.